### PR TITLE
[DOCS-6319] Standard Attribute Updates

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do? What is the motivation?

- Merge duplicate logs and APM entries.
- Link from Span Attributes page to Default Attributes table.
- Removed duplicate content.
- Add more content comparing span attributes and tags.
- Include page in Agent section of TOC.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/trace_collection/tracing_naming_convention/_index.md [hunk 1]
```diff
@@ -14,112 +14,63 @@ further_reading:
...
+
+- [Span tags](#span-tags) are the context around the span.
+- [Span attributes](#span-attributes) are the content of the span.
+
+### Span tags
+
+Span tags are the context around the span. Some examples include:
+
+- **Host tags**: `hostname`, `availability-zone`, `cluster-name`
+- **Container tags**: `container_name`, k`ube_deployment`, `pod_name`
```
Comment (yoannrobin) [inline content/en/tracing/trace_collection/tracing_naming_convention/_index.md:51]:
```suggestion
- **Container tags**: `container_name`, `kube_deployment`, `pod_name`
```

File: content/en/tracing/trace_collection/tracing_naming_convention/_index.md [hunk 2]
```diff
@@ -14,112 +14,63 @@ further_reading:
...
+
+Tags are usually enriched from other data sources like host tags, container tags, or service catalog tags. These tags are added to the span to describe the context. For example, tags might describe the properties of the host and the container the span is coming from, or the properties of the services the span is emitted from.
```
Comment (yoannrobin) [inline content/en/tracing/trace_collection/tracing_naming_convention/_index.md:53]:
```suggestion
Tags are usually enriched from other data sources like tags sourced from host, container, or service catalog. These tags are added to the span to describe the context. For example, tags might describe the properties of the host and the container the span is coming from, or the properties of the services the span is emitted from.
```

File: content/en/tracing/trace_collection/tracing_naming_convention/_index.md [hunk 3]
```diff
@@ -14,112 +14,63 @@ further_reading:
...
+
+One span can have multiple tags with the same key. For example, consider a `team` tag. You might have one `team` tag that describes the team owning the infrastructure layer (host tag), and another `team` tag describing the team owning the service (service catalog tag).
```
Comment (yoannrobin) [inline content/en/tracing/trace_collection/tracing_naming_convention/_index.md:55]: I would get rid of this part. The multiple values with the same key does not work everywhere and can be misleading

File: content/en/tracing/trace_collection/tracing_naming_convention/_index.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Span Tags Semantics
+title: Span Tag Semantics
 kind: documentation
 further_reading:
     - link: 'logs/log_configuration/attributes_naming_convention'
```
Comment (yoannrobin) [inline content/en/tracing/trace_collection/tracing_naming_convention/_index.md:5]: Probably worth adding a link to the naming convention in the further reading links

**Review by yoannrobin — CHANGES_REQUESTED** [review]

**Review by hestonhoffman — APPROVED** [review]
LGTM!

**Review by yoannrobin — APPROVED** [review]

**Pull request merged**